runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -646,17 +646,18 @@ MM_CopyForwardScheme::isObjectInNurseryMemory(J9Object *objectPtr)
 }
 
 MMINLINE void
-MM_CopyForwardScheme::calculateObjectDetailsForCopy(MM_ScavengerForwardedHeader* forwardedHeader, UDATA *objectCopySizeInBytes, UDATA *objectReserveSizeInBytes, bool *doesObjectNeedHash, bool *isObjectGrowingHashSlot)
+MM_CopyForwardScheme::calculateObjectDetailsForCopy(MM_ForwardedHeader* forwardedHeader, UDATA *objectCopySizeInBytes, UDATA *objectReserveSizeInBytes, bool *doesObjectNeedHash)
```

Comment:
At some point we can try to reuse GC_ObjectModelDelegate::calculateObjectDetailsForCopy, but let's keep this change to only what's must be done to deprecate SFH.

Reply:
Sounds good, and you mean keeping `isObjectGrowingHashSlot` right?

Reply:
It's ok to remove it. neither GC_ObjectModelDelegate::calculateObjectDetailsForCopy has it. No further action here. IT was mostly an FYI.